docs/useI18nextProvider.md

```diff
@@ -0,0 +1,151 @@
+---
```

Comment:
I wouldn't add a specific doc chapter about it - the package README should be enough